content/advent-2016/go-timers.md

```diff
@@ -0,0 +1,303 @@
++++
+author = ["Alexander Morozov", "Vyacheslav Bakhmutov"]
+date = "2016-12-04T08:00:00-05:00"
+series = ["Advent 2016"]
+title = "How Do They Do It: Timers in Go"
+draft = true
++++
+
+# How Do They Do It: Timers in Go
+
+This article covers internal implementation of timers in Go. Note that there are
+a lot of links to Go repo in this article, I recommend to follow them to understand
+material better.
+
+## Timers
+
+Timers in Go just do something after a period of time. The user interface for
```

Comment:
Idea was that there are also non-user timers, which users can't use. I think maybe just "Timers are located".